content/topics/file-and-directory-naming/general/_index.md

```diff
@@ -0,0 +1,25 @@
+--- 
+content_type: topic
+ready: true
+title: File and directory naming 
+---
+
+## Naming modules and packages 
+
+Files are refered to as modules and directories as folders or packages.
+
+Naming conventions are rules which enable the titling of electronic and physical folders, documents and records in a consistent and logical way.
+This ensures that the correct records can be located, identified and retrieved from a filing system in a timely fashion, and that they are stored 
+in an appropriate secure location. 
+
+## What should i consider when naming modules and packages
+
+New modules and packages (including third-party frameworks) should adhere to the recommended naming conventions, but if a library already exists that 
+has a different style, internal consistency should take precedence.
+
+## Language specific naming conventions 
+
+##### Python : http://syllabus.africacode.net/topics/file-and-directory-naming/python/
```

Comment:
1. Try use content links to other content as opposed to static links, this has 2 benefits:
    1.1: Thjis make formatting pretty on the frontend
    1.2 this allows the linter to final and highlight any broken links for us to fix
    
2. Is it necessary for the other page to be referenced in this one or will they both be added to learners' boards?